hack/deployer/runner/kind.go

```diff
@@ -207,12 +208,18 @@ func (k *KindDriver) cmd(args ...string) *exec.Command {
 		"ClusterName":     k.plan.ClusterName,
 		"Args":            args,
 	}
+
+	// on mac, the docker socket is located in $HOME
```

Comment:
nit: Mac(intosh) (computer) <> macOS (operating system)
```suggestion
	// on macOS, the docker socket is located in $HOME
```